darts/tests/metrics/test_metrics.py

```diff
@@ -4,6 +4,7 @@
 from darts import TimeSeries
 from darts.metrics import metrics
 from darts.tests.base_test_class import DartsBaseTestClass
+from ...metrics.metrics import quantile_loss
```

Comment:
This should probably be imported from darts.metrics, or metrics directly since it's imported two lines above.